client-react/src/pages/app/change-app-plan/ChangeAppPlanDestinationPlanDetails.tsx

```diff
@@ -192,9 +192,14 @@ export const DestinationPlanDetails: React.FC<DestinationPlanDetailsProps> = ({
       return {
         forbiddenSkus,
         isFunctionApp: isFunctionApp(formProps.values.site),
+        selectedSkuCode: 'EP1',
+        isChangeAppPlan: true,
       };
     }
-    return null;
+    return {
+      selectedSkuCode: 'P2V3',

```

Comment:
Consider changing the default to P1V3 to match the default plan used elsewhere for the Pv3 tier.

```suggestion
      selectedSkuCode: 'P1V3',
```